examples/ensemble/plot_oob_cross_validation.py

```diff
@@ -0,0 +1,87 @@
+"""
+=============================================
+Cheap Cross Validation using Out of Bag Error
+=============================================
+
+Out-of-bag error is an estimation of the generalisation error of an estimator, much
+like the cross validation error. However, it is much cheaper to calculate the out-of-bag
+error, since we can do so with a single training pass of the dataset, as opposed to
+:math:`k` passes in :math:`k`-fold cross validation [1]_.
+
+In this example we perform greedy sequential feature selection with a small number of
+features, and compare the runtime for out-of-bag error and cross validation error.
```

Comment:
This example runs for 33 seconds, that is usually a bit long for an example. I do not think we need to include the `cv=5`.